### ISSUE TITLE
Support to create a new file on a pod with edit

### ISSUE THREAD

**Issue by phochste** [SolidLabResearch/Bashlib#4]
I want to create a new text file on a Pod. Currently I need to create a file first locally, upload it to the pod and then I can further edit it there.

What I expect is something like :  `solid.js --new root:posts/draft.txt` which will create a new resource `${base}/posts/draft.txt`.

Or, a something like:

```
solid.js touch root:posts/draft.txt 
solid.js edit root:posts/draft.txt
```

Which content-type these empty files will get? Need it be provided as option/argument to the cli?

**Comment by Dexagod** [comment]
This should be nothing more than posting an empty file to a URL in the case of touch right?
And then I could add a check to edit to see if the file exists and call the touch function if not

**Comment by phochste** [comment]
Yes something like that. When you touch an existing resource it should do nothing (in unix it updates the timestamp).

**Comment by Dexagod** [comment]
I dont think timestamp updating is possible on Solid. 
I have too much to do atm to work on it, so it will probably be a feature for in 2 weeks :p
In case you'd want to PR yourself, afaik you could just do a fetch(url, {method: PUT}) after a check if the file exists (and should backoff in case of unauthenticated errors as to not overwrite accidentally)
